.github/workflows/build-and-run-model.yaml

```diff
@@ -0,0 +1,36 @@
+# Workflow that builds a Docker image containing the model code,
```

Comment:
This workflow is now the top-level deployment workflow since it calls `build-and-run-batch-job`.